pkgs/tools/X11/find-cursor/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, libX11, libXdamage, libXrender, libXcomposite, libXext, installShellFiles }:
+
+stdenv.mkDerivation {
+  version = "1.6";
+  pname = "find-cursor";
+
+  src = fetchFromGitHub {
+    owner   = "arp242";
+    repo    = "find-cursor";
+    rev     = "45e937a7a334373ee26b6e33341dfb79561d2767";
+    sha256  = "13lpcxklv9ayqapyk9pmwxkinhxah5hkr6n0jc2m5hm68nh220w1";
+  };
+
+
+
+  # This patch help the man page install without makefile script
+  # such that we can shrink the nativeBuildInputs dependency
+  patches = [ ./man.patch ];
```

Comment:
I'm not sure I understand what's going on here, what's wrong with the makefile installation? As far as i can tell that does not add any more dependencies.